verticals/vertical-ats/unifiedModels.ts

```diff
@@ -0,0 +1,62 @@
+import {z} from '@openint/vdk'
+
+export const offer = z.object({
+  id: z.string().uuid(),
+  created_at: z.string().regex(/^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$/),
+  modified_at: z.string().regex(/^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$/),
+  application: z.string().uuid(),
+  closed_at: z.string().regex(/^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$/),
+  sent_at: z.string().regex(/^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$/),
+  start_date: z.string().regex(/^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$/),
+  status: z.string(),
+})
```

Comment:
Let's add an `.openapi` on this

Reply:
`.openapi` is added in the open-sdk definition